daemon/graphdriver/quota/types.go

```diff
@@ -13,4 +14,5 @@ type Control struct {
 	backingFsBlockDev string
 	nextProjectID     uint32
 	quotas            map[string]uint32
+	sync.RWMutex      // protect quotas map
```

Comment:
can you move this above `quotas` ? I think the convention is to put the mutex above the fields that it is protecting

Reply:
moved